api/hex-bot.ts

```diff
@@ -1,12 +1,109 @@
 import { NowRequest, NowResponse } from '@now/node';
 import { Octokit } from '@octokit/rest';
-import Telegraf, { ContextMessageUpdate, Extra } from 'telegraf';
+import Telegraf, { ContextMessageUpdate, Extra} from 'telegraf';
 import { ExtraEditMessage } from 'telegraf/typings/telegram-types';
-import axios from 'axios';
+// import axios from 'axios';
 const fs = require('fs');
+const WizardScene = require('telegraf/scenes/wizard');
```

Comment:
Its better not to use require in codebases as it imports the whole library, fs was an exception because it doesn't contain module.export hence I was forced to use require but it only had 4 functions so not that problematic , try importing the function that you need.

Reply:
Okay. Fair enough.